tests/test_action.py

```diff
@@ -2799,6 +2799,34 @@ def test_moveOnDocument(user: UserClient, user2: UserClient):
     user.post(res=models.Action, data=request_moveOn, status=422)
 
 
```

Comment:
I propose to create at least two more tests.

1. Deactivate a component inside a computer (e.g. a hard disk).

2. Deactivate a device from a user who is not the owner of the device. The test result would have to prove that the system does not allow this case.